mavros/launch/px4_config.yaml

```diff
@@ -114,25 +114,67 @@ distance_sensor:
   hrlv_ez4_pub:
     id: 0
     frame_id: "hrlv_ez4_sonar"
-    orientation: 8 # RPY:{180.0, 0.0, 0.0}
+    orientation: "ROLL_180" # RPY:{180.0, 0.0, 0.0}
     field_of_view: 0.0  # XXX TODO
     send_tf: true
     sensor_position: {x:  0.0, y:  0.0, z:  -0.1}
   lidarlite_pub:
     id: 1
     frame_id: "lidarlite_laser"
-    orientation: 8
+    orientation: "ROLL_180"
     field_of_view: 0.0  # XXX TODO
     send_tf: true
     sensor_position: {x:  0.0, y:  0.0, z:  -0.1}
   sonar_1_sub:
     subscriber: true
     id: 2
-    orientation: 8
+    orientation: "ROLL_180"
   laser_1_sub:
     subscriber: true
     id: 3
-    orientation: 8
+    orientation: "ROLL_180"
+
+## Currently available orientations:
+#  "NONE"
+#  "YAW_45"
+#  "YAW_90"
+#  "YAW_135"
+#  "YAW_180"
+#  "YAW_225"
+#  "YAW_270"
+#  "YAW_315"
+#  "ROLL_180"
+#  "ROLL_180_YAW_45"
+#  "ROLL_180_YAW_90"
+#  "ROLL_180_YAW_135"
+#  "PITCH_180"
+#  "ROLL_180_YAW_225"
+#  "ROLL_180_YAW_270"
+#  "ROLL_180_YAW_315"
+#  "ROLL_90"
+#  "ROLL_90_YAW_45"
+#  "ROLL_90_YAW_90"
+#  "ROLL_90_YAW_135"
+#  "ROLL_270"
+#  "ROLL_270_YAW_45"
+#  "ROLL_270_YAW_90"
+#  "ROLL_270_YAW_135"
+#  "PITCH_90"
+#  "PITCH_270"
+#  "PITCH_180_YAW_90"
+#  "PITCH_180_YAW_270"
+#  "ROLL_90_PITCH_90"
+#  "ROLL_180_PITCH_90"
+#  "ROLL_270_PITCH_90"
+#  "ROLL_90_PITCH_180"
+#  "ROLL_270_PITCH_180"
+#  "ROLL_90_PITCH_270"
+#  "ROLL_180_PITCH_270"
+#  "ROLL_270_PITCH_270"
+#  "ROLL_90_PITCH_180_YAW_90"
+#  "ROLL_90_YAW_270"
+#  "ROLL_315_PITCH_315_YAW_315"
+##
```

Comment:
I think it should have a general Enumerations page like http://wiki.ros.org/mavros/Enumerations. And then we add them all there (there's also others like `MAV_FRAME`).